tests/net/lib/dns_cache/on_target.conf

```diff
@@ -0,0 +1,15 @@
+CONFIG_ZTEST=y
+
+CONFIG_NETWORKING=y
+CONFIG_NET_TEST=y
+CONFIG_NET_L2_ETHERNET=n
+
+CONFIG_MAIN_STACK_SIZE=1344
+CONFIG_DNS_RESOLVER=y
+CONFIG_DNS_RESOLVER_CACHE=y
+CONFIG_LOG=y
```

Comment:
Do we need logs enabled in tests by default? It increases the image size and is not really needed when executing tests. I recall cases from the past, that tests would not fit on some platforms due to logging enabled.

Reply:
Removed log kconfigs